### PR TITLE
Test with Rails 5.2

### PR DESCRIPTION
**Pull request by composerinteralia**
- Add Rails 5.2 to Appraisal
- Avoid specifying tiny Ruby versions to reduce churn on `.travis.yml`

### CODE REVIEW

File: gemfiles/5.2.gemfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "activerecord-jdbcsqlite3-adapter", :platforms => :jruby
+gem "jdbc-sqlite3", :platforms => :jruby
+gem "sqlite3", "~> 1.3.10", :platforms => :ruby
+gem "activerecord", "~> 5.2.0"
+
+gemspec :name => "factory_bot", :path => "../"
```
[the remark this answers is not in the document]
Comment (composerinteralia) [reply gemfiles/5.2.gemfile]: Appraisal generates this file. I will open an issue there.

**Comment by teeparham** [comment]
While specifying only the minor ruby version is valid syntax on Travis, it does _not_ in practice return the latest ruby patch version. For example, `2.4` gets version `2.4.1`, which is not the latest `2.4` vesrion. See line 432 of https://travis-ci.org/thoughtbot/factory_bot/jobs/392869691 . You're also getting `2.2.7` and `2.3.4`. I'm not sure if this is an `rvm` issue or a Travis issue.

**Comment by composerinteralia** [comment]
That's too bad. I guess I was mistaken about Travis having the latest patches pre-installed. I would rather not update `.travis.yml` every couple months for teeny Ruby release. Do you know if there are any good alternatives?

**Comment by teeparham** [comment]
I agree this is a bummer, it has been a problem for years, & I don't know of a solution. I found these related Travis issues, but they are unresolved: 

https://github.com/travis-ci/travis-ci/issues/5294
https://github.com/travis-ci/travis-ci/issues/7077

**Pull request merged**